src/validation_workflow/concrete_validation.py

```diff
@@ -0,0 +1,31 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+from validation_workflow.validation import Validation
+from validation_workflow.validation_args import ValidationArgs
+
+
+class ConcreteValidation(Validation):
```

Comment:
Why do we need this class when Validation already has all abstract methods that can be implemented by all distributions?
https://github.com/opensearch-project/opensearch-build/pull/3038/files#diff-a318e15081b06176764de2b9bd492c632948b9398c4de3b9fc000145d2c18e89R32-R49

Reply:
Hi, @gaiksaya , thanks for asking. There was an error on MyPy when we instantiate the object of the ABC class (validation.py) directly. With a bit research, looks like the ABC class is not meant to be used to create objects directly; hence, we create a concreate subclass of validatioon this way. 
I think mypy has more strick checking; hence, we proposed the concreate class here..
Also, an interesting finding we would love to show you ... if there are only two arguments in RUNNER dictionary, MyPy does not complain .. but if there are more than two, mypy will fail for that reason ...